astropy/table/meta.py

```diff
@@ -165,7 +165,7 @@ def _get_col_attributes(col):
     attrs['name'] = col.info.name
 
     type_name = col.info.dtype.type.__name__
-    if six.PY3 and (type_name.startswith('bytes') or type_name.startswith('str')):
+    if (not six.PY2) and (type_name.startswith('bytes') or type_name.startswith('str')):
```

Comment:
Well the other expression is already within brackets so in this case I think the brackets are just noise.